endpoints.md

```diff
@@ -0,0 +1,108 @@
+## Claim Post
+
+Url: /claim
+
+Method: Any
+
+Accepted JSON fields:
+
+| Field Name      | Required | Content                      |
+|-----------------|----------|------------------------------|
+| api_key         | Yes      | String; the api key          |
+| name            | Yes      | String; the user's name      |
+| is_admin        | No       | Boolean; is an admin or not  |
+| debug           | No       | Int; debug to fixed output   |
+
+## Done Post
+
+Url: /done
+
+Method: Any
+
+Accepted JSON fields:
+
+| Field Name      | Required | Content                      |
+|-----------------|----------|------------------------------|
+| api_key         | Yes      | String; the api key          |
+| post_id         | Yes      | String; the post id          |
+| debug           | No       | Int; debug to fixed output   |
+
+## Unclaim Post
+
+Url: /unclaim
+
+Method: Any
+
+Accepted JSON fields:
+
+| Field Name      | Required | Content                      |
+|-----------------|----------|------------------------------|
+| api_key         | Yes      | String; the api key          |
+| post_id         | Yes      | String; the post id          |
+| debug           | No       | Int; debug to fixed output   |
+
+## Create Keys
+
+Url: /keys/create
+
+Method: Post
+
+Accepted JSON fields:
+
+| Field Name    | Required | Content                      |
+|---------------|----------|------------------------------|
+| api_key       | Yes      | String; the api key          |
+| name          | Yes      | String; the user's name      |
+| is_admin      | No       | Boolean; is an admin or not  |
+| admin_api_key | No       | String; the admin api key    |
```

Comment:
This is the most confusing endpoint in the api and that's because I can't name variables. This particular endpoint only requires `api_key`, `name`, and `is_admin`. The reason for the bad naming is that the endpoint then generates a key and sends it back in the return request so the new key can be given to the user.